src/components/CodeBlockProvider/index.tsx

```diff
@@ -79,5 +79,10 @@ export default function CodeBlockProvider({children}) {
 
 export function useCodeBlockContext() {
   const context = useContext(CodeBlockContext);
+
+  if (!context) {
```

Comment:
When writing tests for this component, I ran into some issues getting the content of the Block to show up. Didn't realize that I needed to use it under the `CodeBlockProvider` and so I thought adding a guard here would be helpful for developers.